src/sage/crypto/stream.py

```diff
@@ -12,16 +12,16 @@
 #                  http://www.gnu.org/licenses/
 #*****************************************************************************
 
-from .cryptosystem import SymmetricKeyCryptosystem
-from .stream_cipher import LFSRCipher, ShrinkingGeneratorCipher
-
-from sage.crypto.util import random_blum_prime
+from sage.arith.misc import GCD as gcd, power_mod
 from sage.monoids.string_monoid import BinaryStrings
-from sage.arith.all import gcd, power_mod
 from sage.rings.finite_rings.finite_field_constructor import FiniteField
 from sage.rings.finite_rings.integer_mod_ring import IntegerModFactory
 from sage.rings.polynomial.polynomial_element import is_Polynomial
 
+from .cryptosystem import SymmetricKeyCryptosystem
```

Comment:
What's the reason for these changes?

Reply:
.all was replaced as described, and I have cleaned up the imports according to best practices (alphabetic, . imports in a separate block)

Reply:
Sorry, I meant the change from "from sage.crypto.util import random_blum_prime" to `.util`.

Reply:
It improves consistency because other modules from this package are already imported in this way.

I made such clean-up changes to the imports in several file swhen the .all replacement already created big diffs.

Reply:
Can we also replace them consistently to absolute imports? There are a few subtle issues with relative imports (e.g. https://stackoverflow.com/questions/16981921/relative-imports-in-python-3) and we do hit them at https://github.com/sagemath/sage/issues/33783.

Reply:
Thanks for reminding me about this problem. Sure, I'll do that.

Reply:
Thanks!

Reply:
OK, done.